docs/developer-docs/latest/setup-deployment-guides/configurations/required/databases.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Database configuration - Strapi Developer Documentation
+description: 
+---
+
+<!-- TODO: update SEO -->
+
+# Database configuration
+
+The `./config/database.js` file is used to define database connections that will be used to store the application content.
+
+:::strapi Supported databases
+The CLI installation guide details [supported database and versions](/developer-docs/latest/setup-deployment-guides/installation/cli.html#preparing-the-installation).
+:::
+
+**Path —** `./config/database.js`.
+
+:::: tabs card
+
+::: tab Bookshelf
```

Comment:
Rename to just something like `SQL or Knex.js` since we don't use bookshelf anymore